dev_tools/spec/lib/solidus/release_drater_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'ostruct'
+require 'solidus/release_drafter'
+
+RSpec.describe Solidus::ReleaseDrafter do
```

Comment:
@waiting-for-dev Spec file name and folder name refers to `release_drater`; should be `release_drafter`.

Reply:
Oh, thanks! Fixed 🙂